client/galaxy/scripts/utils/utils.js

```diff
@@ -349,6 +349,40 @@ export function setWindowTitle(title) {
     }
 }
 
+/**
+ * Calculate a 32 bit FNV-1a hash
+ * Found here: https://gist.github.com/vaiorabbit/5657561
+ * Ref.: http://isthe.com/chongo/tech/comp/fnv/
+ *
+ * @param {string} str the input value
+ * @returns {integer}
+ */
+function hashFnv32a(str) {
+    /*jshint bitwise:false */
+    var i, l,
+        hval = 0x811c9dc5;
+
+    for (i = 0, l = str.length; i < l; i++) {
+        hval ^= str.charCodeAt(i);
+        hval += (hval << 1) + (hval << 4) + (hval << 7) + (hval << 8) + (hval << 24);
+    }
+    return hval >>> 0;
+}
+
+export function generateTagColor(tagValue) {
+    //var r = (15 - ((hash & 0xf) % 6)).toString(16),
```

Comment:
drop?

Reply:
This was the 'light' version, yes, should be dropped before merge.